tests/phpunit/tests/http/base.php

```diff
@@ -32,7 +32,7 @@ function skipTestOnTimeout( $response ) {
 			$this->markTestSkipped( 'HTTP timeout' );
 		}
 
-		if ( 0 === strpos( $response->get_error_message(), 'stream_socket_client(): unable to connect to tcp://s.w.org:80' ) ) {
+		if ( 0 === strpos( $response->get_error_message(), 'stream_socket_client(): unable to connect to tcp://twemoji.classicpress.net:80' ) ) {
```

Comment:
This existing code is 7 different kinds of wrong.  I think we can leave it as-is for this issue/PR though, see https://github.com/ClassicPress/ClassicPress/issues/25 for a better way to address it.